### PR TITLE
London Class 7 - Yasemin Bayraktutan-GIT and GITHUB-WEEK2

### PR DESCRIPTION
**Pull request by ybayraktutan**
# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -44,67 +44,70 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
+JavaScript is the Programming Language for the Web
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+- It helps people to share code/information,
+- It used to keep the history of changes, 
+- It makes collaboration easier.
```
Comment (omarsharif-gr) [inline homework.md:55]:
```suggestion
- It helps people to share code/information,
- It used to keep the history of changes, 
- It makes collaboration easier.
```
This is great - well done! One thing you can improve on is whether there are any other benefits of Git/GitHub. The first and last points are very similar. Great job!

**Comment by ademalagoz** [comment]
Hi Yasemin
Great job with the homework, I just made one suggestion in case you want to use it.

File: homework.md (hunk 1 above)
Comment (ybayraktutan) [reply homework.md:55]: Thank you very much

File: homework.md [hunk 2]
```diff
@@ -44,67 +44,70 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
+VS Code
 
 ## Question 17
 
 What do you use "Commit Messages" for?
 
-To record what changes you made to the code
+To give clear, helpful messages, describing what changed, and why
 
 ## Question 18
```
[the remark this answers is not in the document]
Comment (ybayraktutan) [reply homework.md:112]: Thank you very much. I think original answer is correct.

**Pull request closed without merging**